includes/widgets/text-editor.php

```diff
@@ -95,13 +95,17 @@ public function get_keywords() {
 	 *
 	 * Retrieve the list of style dependencies the widget requires.
 	 *
+	 * The 'widget-text-editor' style is required only when the drop cap is used.
+	 * Therefor, style should not be loaded on the widget level, rather only on
+	 * control level when the drop cap is active.
+	 *
 	 * @since 3.24.0
 	 * @access public
 	 *
 	 * @return array Widget style dependencies.
 	 */
 	public function get_style_depends(): array {
```

Comment:
Maybe remove this function completely?

Reply:
Yes, initially I did, but then I thought that we need to keep track on the changes so that in the future developers won't remove this optimization.

Reply:
You can decide about it.

Reply:
Let's keep it.